README.md

```diff
@@ -1,6 +1,27 @@
 # EuroPi
 
-The EuroPi is a fully user reprogrammable module based on the [Raspberry Pi Pico](https://www.raspberrypi.com/products/raspberry-pi-pico/), which allows users to process inputs and controls to produce outputs based on code written in Python. The entire project is open-source.
+
+### The EuroPi is a fully user reprogrammable module based on the [Raspberry Pi Pico](https://www.raspberrypi.com/products/raspberry-pi-pico/), which allows users to process inputs and controls to produce outputs based on code written in Python. The entire project is open-source.
+
+---
+
+<p>
```

Comment:
[optional] Most repos will start with the main title, immediately followed by the shields, then the summary paragraph. It's fine in this order too though if you prefer.